enterprise/users/src/main/java/io/crate/operation/user/Privileges.java

```diff
@@ -48,6 +49,13 @@ static void ensureUserHasPrivilege(Privilege.Clazz clazz,
                                        String ident,
                                        User user) throws MissingPrivilegeException {
         assert user != null : "User must not be null when trying to validate privileges";
+
+        // information_schema should not be protected
+        if (!Privilege.Clazz.CLUSTER.equals(clazz) &&
+            ident != null &&
+            ident.startsWith(InformationSchemaInfo.NAME)) {
```

Comment:
I think we should extract the schema name and use equals to test here. Currently, a `doc` table called `information_schema_table` will be unprotected.